packages/bridging/src/BridgingSdk/BridgingSdk.test.ts

```diff
@@ -240,5 +241,215 @@ adapterNames.forEach((adapterName) => {
+        })
+
+        expect(results).toHaveLength(2)
+        expect(results[0]?.providerDappId).toBe('mockProvider')
+        expect(results[1]?.providerDappId).toBe('cow-sdk://bridging/providers/mock3')
+
+        // All should be successful
+        results.forEach(result => {
+          expect(result.quote).toBeTruthy()
+          expect(result.error).toBeUndefined()
+        })
+
+        // Verify only requested providers were called
+        expect(mockProvider.getQuote).toHaveBeenCalled()
+        expect(mockProvider2.getQuote).not.toHaveBeenCalled()
+        expect(mockProvider3.getQuote).toHaveBeenCalled()
+      })
+
+      it('should handle provider errors gracefully', async () => {
+        const error = new Error('Provider failed')
+        mockProvider2.getQuote = jest.fn().mockRejectedValue(error)
+
+        const results = await bridgingSdk.getMultiQuotes({
+          quoteBridgeRequest,
+        })
+
+        expect(results).toHaveLength(3)
+
+        // First provider should succeed
+        expect(results[0]?.providerDappId).toBe('mockProvider')
+        expect(results[0]?.quote).toBeTruthy()
+        expect(results[0]?.error).toBeUndefined()
+
+        // Second provider should fail
+        expect(results[1]?.providerDappId).toBe('cow-sdk://bridging/providers/mock2')
+        expect(results[1]?.quote).toBeNull()
+        expect(results[1]?.error).toBeTruthy()
+        expect(results[1]?.error?.message).toContain('Provider failed')
+
+        // Third provider should succeed
+        expect(results[2]?.providerDappId).toBe('cow-sdk://bridging/providers/mock3')
+        expect(results[2]?.quote).toBeTruthy()
+        expect(results[2]?.error).toBeUndefined()
+      })
+
+      it('should throw error for same-chain requests', async () => {
+        const sameChainRequest = {
+          ...quoteBridgeRequest,
+          buyTokenChainId: quoteBridgeRequest.sellTokenChainId,
+        }
+
+        await expect(
+          bridgingSdk.getMultiQuotes({
+            quoteBridgeRequest: sameChainRequest,
+          })
+        ).rejects.toThrow(
+          'getMultiQuotes() is only for cross-chain bridging. For single-chain swaps, use getQuote() instead.'
+        )
+      })
+
+      it('should throw error for unknown provider dappId', async () => {
+        await expect(
+          bridgingSdk.getMultiQuotes({
+            quoteBridgeRequest,
+            providerDappIds: ['unknown-provider'],
+          })
+        ).rejects.toThrow(
+          "Provider with dappId 'unknown-provider' not found. Available providers: mockProvider, cow-sdk://bridging/providers/mock2, cow-sdk://bridging/providers/mock3"
+        )
+      })
+
+      it('should pass advanced settings to bridge quotes', async () => {
+        const advancedSettings: SwapAdvancedSettings = {
+          quoteRequest: {
+            validTo: Math.floor(Date.now() / 1000) + 3600, // 1 hour from now
```

Comment:
What about setting a constant? Even if in the past, wouldn't matter since the call is mocked.

Reply:
Makes sense!
https://github.com/cowprotocol/cow-sdk/pull/518/commits/3e5f7ab830091a51b1d3b6ad70a4224ff7cd7c28